models/iaf_psc_alpha.h

```diff
@@ -40,71 +40,98 @@ namespace nest
+
+The synaptic input current has an excitatory and an inhibitory component
+
+.. math::
+
+   I_{\text{syn}}(t) = I_{\text{syn, ex}}(t) + I_{\text{syn, in}}(t)
+
+where
+
+.. math::
 
-   The present implementation uses individual variables for the
-   components of the state vector and the non-zero matrix elements of
-   the propagator. Because the propagator is a lower triangular matrix,
-   no full matrix multiplication needs to be carried out and the
-   computation can be done "in place", i.e. no temporary state vector
-   object is required.
+   I_{\text{syn, ex}}(t) = \sum_{(t_m, w_m)\\w_m\geq 0} w_m i_{\text{syn, ex}}(t-t_m)
 
-   The template support of recent C++ compilers enables a more succinct
-   formulation without loss of runtime performance already at minimal
-   optimization levels. A future version of iaf_psc_alpha will probably
-   address the problem of efficient usage of appropriate vector and
-   matrix objects.
+is the sum over all incoming excitatory spikes, with a corresponding expression for inhibitory spikes (:math:`w_m < 0`).
+Finally, individual post-synaptic currents are given by
+
+.. math::
+
+   i_{\text{syn, X}}(t) = \frac{e}{\tau_{\text{syn, X}}} t e^{-\frac{t}{\tau_{\text{syn, X}}}} \Theta(t)
+
+where :math:`\Theta(x)` is the Heaviside step function. The PSCs are normalized to unit maximum, i.e.,
+
+.. math::
+
+   i_{\text{syn, X}}(t) = 1 \Leftrightarrow t = \tau_{\text{syn, X}} \;.
+
+As a consequence, the total charge transferred by a single PSC depends
+on the synaptic time constant according to
+
+.. math::
+
+   \int_0^{infty}  i_{\text{syn, X}}(t) dt = e \tau_{\text{syn, X}} \;.
+
+By default, :math:`V_m` is not bounded from below. To limit
+hyperpolarization to biophysically plausible values, set parameter
+:math:`V_{\text{min}}` as lower bound of :math:`V_m`.
 
 .. note::
 
-   If ``tau_m`` is very close to ``tau_syn_ex`` or ``tau_syn_in``, the model
-   will numerically behave as if ``tau_m`` is equal to ``tau_syn_ex`` or
-   ``tau_syn_in``, respectively, to avoid numerical instabilities.
+   NEST uses exact integration [1]_, [2]_ to integrate subthreshold membrane
+   dynamics with maximum precision; see also [3]_.
+
+   If :math:`\tau_m\approx \tau_{\text{syn, ex}}` or
+   :math:`\tau_m\approx \tau_{\text{syn, in}}`, the model will
+   numerically behave as if :math:`\tau_m = \tau_{\text{syn, ex}}` or
+   :math:`\tau_m = \tau_{\text{syn, in}}`, respectively, to avoid
+   numerical instabilities.
 
    For implementation details see the
    `IAF_neurons_singularity <../model_details/IAF_neurons_singularity.ipynb>`_ notebook.
 
-See also [3]_.
 
 Parameters
 ++++++++++
 
 The following parameters can be set in the status dictionary.
 
-=========== ======  ==========================================================
+=========== ======  =======================================================================
```

Comment:
Should we add a title to this table? Are these membrane parameters?

Reply:
There are membrane parameters, various other neuron parameters and also synaptic parameters. I think just "Parameters" is the most fitting title.